src/vs/code/electron-main/app.ts

```diff
@@ -1471,4 +1482,49 @@ export class CodeApplication extends Disposable {
 		// Refs: https://github.com/microsoft/vscode/issues/234064
 		validatedevDeviceId(this.stateService, this.logService);
 	}
+
+	private async moveToApplicationsFolder(accessor: ServicesAccessor): Promise<void> {
+		const dialogMainService = accessor.get(IDialogMainService);
+
+		const { response } = await dialogMainService.showMessageBox({
+			type: 'info',
+			buttons: [
+				localize({ key: 'move', comment: ['&& denotes a mnemonic'] }, "&&Move to Applications folder"),
+				localize({ key: 'doNotMove', comment: ['&& denotes a mnemonic'] }, "&&Cancel")
+			],
+			message: localize('moveToApplicationsFolderWarning', "Current version of {0} is installed outside the Applications folder, would you like to move it ?", this.productService.nameLong),
+			checkboxLabel: localize('remember', "Do not ask again"),
```

Comment:
You will have to implement this actually, this is just the UI part of it.

Reply:
Should be removed, this was left over from previous iteration.